subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/TravisPlugin.java

```diff
@@ -74,5 +79,12 @@ public void execute(ReleaseNeededTask t) {
                 t.setPullRequest(isPullRequest);
             }
         });
+
+        GitCommitTask gitCommit = (GitCommitTask) project.getTasks().getByName(GitPlugin.GIT_COMMIT_TASK);
+        //can we construct the url from some Travis env variable?
+        //this needs to be safe for local invocation where there are no travis env variables.
+        String travisJobUrl = "https://travis-ci.org/mockito/shipkit/builds/288214072";
+        String postfix = ". CI job: " + travisJobUrl + " " + conf.getGit().getCommitMessagePostfix();
```

Comment:
The first commit will be too long. It would be better to have extra information available in the second (and the following) lines.

Reply:
Probably also with the information about the release notes generation.

Reply:
Good idea!